README.md

```diff
@@ -1,12 +1,34 @@
 # News Site
 
-Replace this readme with your own information about your project. 
+The assignment is to make a news page using flexbox and making it responsive.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I've added an animation on the logo, box-shadow including a discreet hover effect on the cards.
+The navbas is sticky.
 
+https://stackoverflow.com/c/technigo/questions/1952
+
+https://stackoverflow.com/c/technigo/questions/1970
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+I attacked the challenge in a fairly practical matter and had a functioning page pretty fast.
+My biggest issue was that I didn't take time beforehand to make a design to follow when coding.
```

Comment:
Sometimes finding a design that you want to imitate/use as inspiration can be useful in these situations 👍